.github/workflows/ci-nix.yml

```diff
@@ -332,3 +330,39 @@ jobs:
         webhook: ${{ secrets.DISCORD_WEBHOOK }}
         # current job is a success, but that's not what we're interested in
         status: failure
+
+  deb-binary:
```

Comment:
We should probably do this in `Release packages` workflow, as both will be building release builds of stuff, so possibly can re-use most of the work.

Reply:
I've added the `Release packages` to the artifact upload.
Is that what you meant?

Reply:
Nope.

There is already a job "Release packages" https://github.com/fedimint/fedimint/blob/8864e239e539c716b4f970397f508598dbb1c383/.github/workflows/ci-nix.yml#L281 and I think under the hood is already has to build the same thing (release versions of packages). Bundling into deb/rpm is just an extra step.

So by doing all the stuff here in that job, we could avoid doing the same thing twice.